Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -121,3 +121,4 @@
 --rules blankLinesBetweenChainedFunctions
 --rules unusedPrivateDeclaration
 --rules emptyExtension
+--rules preferCountWhere
```

Comment:
Does SwiftFormat ignore unknown options? I'm asking to learn.

Reply:
No, it will fail with an "unknown rule" error. At the time I left this comment CI on this PR was failing. Then later I released `0.56-beta-3` which included the new rule, and updated PR.